runners/spark/src/main/java/org/apache/beam/runners/spark/metrics/AggregatorMetric.java

```diff
@@ -16,15 +16,14 @@
  * limitations under the License.
  */
 
-package org.apache.beam.runners.spark.aggregators.metrics;
+package org.apache.beam.runners.spark.metrics;
 
 import com.codahale.metrics.Metric;
 
 import org.apache.beam.runners.spark.aggregators.NamedAggregators;
 
 /**
- * An adapter between the {@link NamedAggregators} and codahale's {@link Metric}
- * interface.
+ * An adapter between the {@link NamedAggregators} and Codahale's {@link Metric} interface.
```

Comment:
This implementation of Codahale's `Metric` interface serves as a glue to be able to report aggregators (and metrics, in a different class in this commit `SparkBeamMetric`) using `Spark`'s support of metric sinks.